cookbooks/pelias/attributes/external.rb

```diff
@@ -3,7 +3,6 @@
 
 # nodejs
 default[:nodejs][:version]                      = '4.4.7'
-default[:nodejs][:checksum_linux_x64]           = '4e1578efc2a2cc67651413a05ccc4c5d43f6b4329c599901c556f24d93cd0508'
```

Comment:
Hey @pmezard,
I tried the change you suggested, and while it does indeed work, if I change the checksum to an incorrect, but valid checksum, it still installs nodejs. This suggests to me that the checksum isn't being used. I tried a bunch of combinations of attributes that look correct, and was never able to make the installation of nodejs fail because of a checksum error. So I think something is actually broken in the chef recipe for nodejs, but that's definitely outside my skill level with chef to fix. Its sad, but I don't know any solution currently.